modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -192,7 +198,106 @@ $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_worker-machi
 ----
 +
 Because you create and manage the worker machines yourself, you do not need to initialize these machines.
+//The current AWS UPI method removes all of the `99_openshift-cluster-api_worker-machineset-* files. What is different about the local zones that means that we should retain them? Should we remove whatever the installation program creates and then specifically create this new file?
 endif::aws,azure,ash,gcp[]
+ifdef::localzone[]
+. Create the machineset for the nodes in your local zone.
+.. Export a variable to define the instance type for your control plane machines by running the following command:
```

Comment:
```suggestion
. Create the MachineSet manifests for the worker nodes in your Local Zone.
.. Export a variable to define the instance type for the worker machine to deploy on the Local Zone subnet by running the following command:
```

Reply:
I've updated the step text but retained the styling of "machine set" to match the [repo guidelines on API objects](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references).